source/rafcon/gui/utils/splash_screen.py

```diff
@@ -45,18 +46,25 @@ def __init__(self, width=530, height=350, contains_image=False):
         if contains_image:
             main_vbox.pack_start(self.image, True, True, 0)
 
-        # add label to display text, the text can be changed by the text() method.
-        # Align it in the middle of the gtk window
-        self.label = Gtk.Label(label="")
-        self.label.set_xalign(0.5)
-        self.label.set_yalign(0.5)
-        main_vbox.pack_start(self.label, False, True, 10)
-        main_vbox.set_spacing(0)
+        if is_custom_design_enabled() and not global_design_config.get_config_value("SPLASH_SCREEN_SHOW_TEXT", True):
```

Comment:
Is this check for custom design really needed? And if so, what for?

Reply:
No, it wasn't needed.